tests/TestCli.cpp

```diff
@@ -983,6 +996,59 @@ void TestCli::testGenerate()
     QCOMPARE(m_stderrFile->readLine(), QByteArray("Invalid password length bleuh\n"));
 }
 
+void TestCli::testImport()
+{
+    Import importCmd;
+    QVERIFY(!importCmd.name.isEmpty());
+    QVERIFY(importCmd.getDescriptionLine().contains(importCmd.name));
+
+    QScopedPointer<QTemporaryDir> testDir(new QTemporaryDir());
+    QString databaseFilename = testDir->path() + "testImport1.kdbx";
+
+    Utils::Test::setNextPassword("a");
+    importCmd.execute({"import", m_xmlFile->fileName(), databaseFilename});
+
+    m_stderrFile->reset();
+    m_stdoutFile->reset();
+
+    QCOMPARE(m_stdoutFile->readLine(),
+             QByteArray("Insert password to encrypt database (Press enter to leave blank): \n"));
+    QCOMPARE(m_stdoutFile->readLine(), QByteArray("Successfully imported database.\n"));
+
+    Utils::Test::setNextPassword("a");
+    auto db = QSharedPointer<Database>(Utils::unlockDatabase(databaseFilename, true, "", "", Utils::DEVNULL));
+    QVERIFY(db);
+    auto* entry = db->rootGroup()->findEntryByPath("/Sample Entry 1");
+    QVERIFY(entry);
+    QCOMPARE(entry->username(), QString("User Name"));
+
+    // Should refuse to create the database if it already exists.
+    qint64 pos = m_stdoutFile->pos();
+    qint64 errPos = m_stderrFile->pos();
+    importCmd.execute({"import", m_xmlFile->fileName(), databaseFilename});
+    m_stdoutFile->seek(pos);
+    m_stderrFile->seek(errPos);
+    // Output should be empty when there is an error.
+    QCOMPARE(m_stdoutFile->readAll(), QByteArray(""));
+    QString errorMessage = QString("File " + databaseFilename + " already exists.\n");
+    QCOMPARE(m_stderrFile->readAll(), errorMessage.toUtf8());
+
+    m_stdoutFile->reset();
+
+    // Quiet option
+    QScopedPointer<QTemporaryDir> testDirQuiet(new QTemporaryDir());
+    QString databaseFilenameQuiet = testDirQuiet->path() + "testImport2.kdbx";
+
+    pos = m_stdoutFile->pos();
+    Utils::Test::setNextPassword("a");
+    importCmd.execute({"import", "-q", m_xmlFile->fileName(), databaseFilenameQuiet});
+    m_stdoutFile->seek(pos);

```

Comment:
@jsachs here we're not actually testing that there was no output on stdout with the quiet option. And it would probably fail since it doesn't seem to be implemented.

Should be fairly easy, you just have to defined the output stream as
```
TextStream outputTextStream(parser->isSet(Command::QuietOption) ? Utils::DEVNULL : Utils::STDOUT, QIODevice::WriteOnly);
```